internal/cypress/config.go

```diff
@@ -211,6 +225,18 @@ func ValidateCypressConfiguration(configFile string) error {
 	return nil
 }
 
+func isCypressCfgIgnored(sauceIgnoreFile string) (bool, error) {
+	if _, err := os.Stat(sauceIgnoreFile); err != nil {
+		return false, nil
+	}
+	matcher, err := sauceignore.NewMatcherFromFile(sauceIgnoreFile)
+	if err != nil {
+		return false, err
+	}
+
+	return matcher.Match([]string{"cypress.json"}, false), nil
```

Comment:
The cypress.json is also configurable :D You have to take the filename from our config 💡

Reply:
fixed!